xml/System.Diagnostics/DiagnosticListener.xml

```diff
@@ -213,10 +213,10 @@ If setup for the notification is expensive, you can call `IsEnabled` before perf
         <Parameter Name="arg2" Type="System.Object" Index="2" FrameworkAlternate="netcore-2.0;netcore-2.1;netcore-2.2;netcore-3.0" />
       </Parameters>
       <Docs>
-        <param name="name">To be added.</param>
-        <param name="arg1">To be added.</param>
-        <param name="arg2">To be added.</param>
-        <summary>Checks if the DiagnosticListener is enabled.</summary>
+        <param name="name">Name of the event to check if there are any subscribers</param>
+        <param name="arg1">Object that represents a context.</param>
+        <param name="arg2">Object that represents a context.</param>
+        <summary>Checks if the DiagnosticListener is enabled via a delegate provided by a subscriber through the Subscribe call.</summary>
```

Comment:
I think we could either say "this instance" as suggested, or maybe add a link to the `DiagnosticListener` type, although we are already there, so maybe there's no point.
```suggestion
        <summary>Checks if this instance is enabled via a delegate provided by a subscriber through the <see cref="M:System.Diagnostics.DiagnosticListener.Subscribe%2A" /> call.</summary>
```